src/views/messages/presentation.jsx

```diff
@@ -140,14 +140,10 @@ var SocialMessagesList = React.createClass({
             />;
         }
     },
-    renderSocialMessages: function (messages, unreadCount) {
+    renderSocialMessages: function (messages) {
```

Comment:
It's nice that this simplifies the rest of the code and unreadCount doesn't get passed around everywhere 👍